runtimes/common/Cargo.toml

```diff
@@ -21,17 +21,17 @@ kilt-asset-dids.workspace = true
 scale-info = {workspace = true, features = ["derive"]}
 smallvec.workspace = true
 
-# Internal dependencies
-attestation.workspace = true
-ctype.workspace = true
-delegation = {workspace = true, optional = true}
-did = {workspace = true, optional = true}
-pallet-did-lookup = {workspace = true, optional = true}
-pallet-inflation = {workspace = true, optional = true}
+# # Internal dependencies
```

Comment:
Why these comments?

Reply:
I was thinking it is nice to separate internal and external dependencies with a comment. But I removed it now in my newest commit.